packages/compiler-cli/src/ngtsc/annotations/src/util.ts

```diff
@@ -351,3 +351,28 @@ export function readBaseClass(
 
   return null;
 }
+
+const parensWrapperTransformerFactory: ts.TransformerFactory<ts.Expression> =
+    (context: ts.TransformationContext) => {
```

Comment:
NIT: the type is not needed on this parameter because it is implied by the type of `parensWrapperTransformerFactory`.

```suggestion
    context => {
```

Reply:
I prefer explicit return types regardless.

Reply:
@alxhub did you mean argument type?